docs/deploy-with-olm.md

```diff
@@ -109,6 +109,37 @@ the Block CR, you may run in to issues where the pod reports being unschedulable
 
 In this example, the disks were prevsiously used, and a filesystem was applied.  Be sure to either use fresh disks, or remove any partitions and file systems if you're setting up a Block mode CR.
 
+### Create a CR using Tolerations
+
+The following example assumes that some nodes in the cluster have [taints](https://docs.openshift.com/container-platform/latest/nodes/scheduling/nodes-scheduler-taints-tolerations.html) that designate them for use with the local-storage Operator.
+
+For instance, assume that some nodes have a key, `localstorage`, with a value of `true`. With the following configuration the local-storage Operator will only search for volumes on nodes that match this key, or nodes that are generally available for scheduling:
+
+```yaml
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: "local-disks"
+  namespace: "local-storage"
+spec:
+  tolerations:
+    - key: localstorage
+      operator: Equal
+      value: true
+  storageClassDevices:
+    - storageClassName: "local-sc"
+      volumeMode: Filesystem
+      fsType: xfs
+      devicePaths:
+        - /dev/xvdf
+```
+
+The defined tolerations will be passed to the resulting DaemonSets, and will determine the available nodes for the diskmaker and provisioner pods.
+
+#### NOTE about Tolerations and NodeSelector
+
+When using both a NodeSelector and Tolerations only nodes that match both sets will be used by the local-storage Operator. Be careful with the union of these sets, as it may be more restrictive than intended.
+
```

Comment:
Same as above - toleration merely say that pods created by the operator will tolerate the specified taint if node has the same taint. I do not think they are equivalent to selectors.

Reply:
I've removed this note entirely.